packages/dds/tree/src/test/utils.ts

```diff
@@ -890,23 +890,22 @@ export function makeEncodingTestSuite<TDecoded, TEncoded, TContext>(
 					: codec.json;
 			describe("can json roundtrip", () => {
 				for (const includeStringification of [false, true]) {
-					describe(
-						includeStringification ? "with stringification" : "without stringification",
-						() => {
-							for (const [name, data, context] of encodingTestData.successes) {
-								it(name, () => {
-									// eslint-disable-next-line @typescript-eslint/no-non-null-assertion
-									let encoded = jsonCodec.encode(data, context!);
-									if (includeStringification) {
-										encoded = JSON.parse(JSON.stringify(encoded));
-									}
-									// eslint-disable-next-line @typescript-eslint/no-non-null-assertion
-									const decoded = jsonCodec.decode(encoded, context!);
-									assertEquivalent(decoded, data);
-								});
-							}
-						},
-					);
+					describe(includeStringification
+						? "with stringification"
+						: "without stringification", () => {
+						for (const [name, data, context] of encodingTestData.successes) {
```

Comment:
Hmm I'm torn about this one. The ones before where the `function()` declaration ended up at the end of a line didn't feel so bad for some reason, then I hit this and it felt weird, and now the other ones also feel a bit weird 🤔 . Is it configurable by any chance?

Reply:
The only config we can do to influence this (maybe) is to apply a different line length to this file, but that would have other side effects. If we really don't like the changes in this file, we could use inline ignores I think.

Reply:
I don't think I'd bother if it's just me. If someone else has similar opinion maybe we can think about it.

Reply:
The new line wrapping here seems fine, though I also preferred the old one I don't think its worth worrying about.

I do find it odd that the line 896 has the same indent as the one before it despite being inside a new level of nesting from the lambda. To me that looks like a biome bug which should probably be reported.

My suggestion is to go ahead with merging this update (assuming everything else is fine) but report this case as a bug.

Reply:
Also the choice to line wrap one member of an argument list, and put part of the next member on the same line as the last part of the first one seems odd. I'd expect putting each argument on its own line if following an argument that was line wrapped. which is how the old version worked.

Reply:
After finding a lot for cases of this same issue, but worse, I'm no longer sure we want to land this update as is. If it does bug fixes we need, then maybe, this it makes a lot of test bodies really hard to seperate from the line wrapped descriptions

Reply:
I added inline formatter ignore statements to the most egregious problems in test files. I linked to the relevant Biome issue in the ignore statement.